tests/e2e/default/enketo/pregnancy-visit.wdio-spec.js

```diff
@@ -26,6 +27,26 @@ describe('Pregnancy Visit', () => {
     await loginPage.login(offlineUser);
   });
 
+  afterEach(async () => {
+    await commonPage.goToReports();
+    const reports = await reportsPage.reportsListDetails();
+    for (const report of reports) {
+      if (report.heading === pregnantWoman.name) {
+        await utils.deleteDocs([report.dataId]);
+      }
+    }
```

Comment:
Can we somehow make this more robust? Like save the doc ids when they are created (somehow) instead of going over this visible list after the test?